libs/ledger/include/ledger/dag/dag_node.hpp

```diff
@@ -175,13 +173,12 @@ struct MapSerializer<ledger::DAGNode, D>
   static uint8_t const TYPE                    = 1;
   static uint8_t const PREVIOUS                = 2;
   static uint8_t const CONTENTS                = 3;
-  static uint8_t const CONTRACT_DIGEST         = 4;
-  static uint8_t const CONTRACT_ADDRESS        = 5;
-  static uint8_t const IDENTITY                = 6;
-  static uint8_t const HASH                    = 7;
-  static uint8_t const SIGNATURE               = 8;
-  static uint8_t const OLDEST_EPOCH_REFERENCED = 9;
-  static uint8_t const WEIGHT                  = 10;
+  static uint8_t const CONTRACT_ADDRESS        = 4;
+  static uint8_t const IDENTITY                = 5;
+  static uint8_t const HASH                    = 6;
+  static uint8_t const SIGNATURE               = 7;
+  static uint8_t const OLDEST_EPOCH_REFERENCED = 8;
+  static uint8_t const WEIGHT                  = 9;
```

Comment:
There's a call to `map_constructor` below which should now receive 9 instead of 10.